src/Theme/Colour.stories.tsx

```diff
@@ -1,19 +1,14 @@
-import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
-import { withKnobs, text, boolean, select } from '@storybook/addon-knobs';
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
 import { DisplayBox, DisplayBoxWrapper } from '../DisplayBox';
 import { DisplayBoxTitle, DisplayBoxSubTitle, DisplayBoxBody } from '../DisplayBox/DisplayBox.styles.ts';
 
-<Meta title="Theme|Colour" />
+export default {
+  title: 'Theme/Colours',
+  // component: Button,
+} as Meta;
 
-# Colours
-
-This is the colours that will be used throught all edozo apps (customer facing). The design files can be found [here](https://www.figma.com/file/OpCqZzeuBbb8NBshcsBRpc/Edozo-maps-DS?node-id=155%3A1235).
-
-### Primary
-
-This colour is refered to as **Sky Blue**, it is aliased to **Primary** within the theme for ease of use and consistency
-
-<Story name="Primary">
+export const Primary: Story = () => (
   <DisplayBoxWrapper>
     <DisplayBox background="skyBlueLighter">
```

Comment:
I get a typesript warning here 
<img width="445" alt="Screenshot 2020-10-06 at 11 46 04" src="https://user-images.githubusercontent.com/30238668/95192692-7a34c900-07ca-11eb-829a-14ede741a4c5.png">

Reply:
What was this one, just out of curiosity?

Reply:
When I changed DisplayBox to use `React.FC<StyleProps>` it pulled through the correct type info